### PR TITLE
e2e installer test

### PR DESCRIPTION
**Pull request by ilackarms**
wires up the install syncer into main, contains an e2e test

old e2e tests were moved to e2e2

### CODE REVIEW

**Comment by ilackarms** [comment]
/kick
i beleive this failed because of issues running in parallel

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		meshClient := helpers.MustMeshClient()
+		Eventually(func() error {
+			_, err := meshClient.Read("supergloo-system", "my-istio", clients.ReadOpts{})
+			return err
+		}).ShouldNot(HaveOccurred())
+
+		err = utils.Supergloo("uninstall --name=my-istio")
+		Expect(err).NotTo(HaveOccurred())
+
+		Eventually(func() error {
```
Comment (rickducott) [inline test/e2e/e2e_test.go]: This func() is copy/pasted from above and can be extracted into a shared thing

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			_, err := kube.CoreV1().Services("istio-system").Get("istio-pilot", metav1.GetOptions{})
+			return err
+		}, time.Minute*2).Should(HaveOccurred())
+
+		Eventually(func() error {
+			_, err := meshClient.Read("supergloo-system", "my-istio", clients.ReadOpts{})
```
Comment (rickducott) [inline test/e2e/e2e_test.go]: same with this

File: test/e2e/e2e_suite_test.go [hunk 3]
```diff
@@ -1,27 +1,88 @@
...
-	Expect(err).ShouldNot(HaveOccurred())
+	kube = testutils.MustKubeClient()
+	// create sg ns
+	_, err := kube.CoreV1().Namespaces().Create(&v1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{Name: "supergloo-system"},
+	})
+	Expect(err).NotTo(HaveOccurred())
+
+	// start supergloo
+	go func() {
```
Comment (rickducott) [inline test/e2e/e2e_suite_test.go]: Should we clean this up after the suite?

**Pull request merged**